model/pagoDAO.php

```diff
@@ -58,16 +55,59 @@ public function delete($id)
 
     public function read()
     {
-        $id_escuela = $_SESSION['id_escuela'];
         require_once 'pagoDTO.php';
         $query = "SELECT * FROM pago
         INNER JOIN cobro on pago.id_cobro=cobro.id_cobro
         INNER JOIN alumno on cobro.id_alumno=alumno.id_alumno
         INNER JOIN escuela on alumno.id_escuela=escuela.id_escuela
         INNER JOIN director on escuela.id_escuela=director.id_escuela 
-        WHERE pago.id_cobro=cobro.id_cobro and cobro.id_alumno=alumno.id_alumno and alumno.id_escuela = escuela.id_escuela and director.id_escuela = escuela.id_escuela and director.id_escuela and director.id_escuela = '" . $id_escuela . "'";
+        WHERE pago.id_cobro=cobro.id_cobro 
+        and cobro.id_alumno=alumno.id_alumno 
+        and alumno.id_escuela = escuela.id_escuela 
+        and director.id_escuela = escuela.id_escuela
+        and director.id_escuela 
```

Comment:
Ese where está mal no creo que funcione revisalo

Reply:
Si funciona consulta los pagos bien.

Reply:
Revisa la consulta está bien??
and director.id_escuela????